backend/leaderboard/views.py

```
@@ -60,7 +60,7 @@ class LeaderboardViewSet(viewsets.ReadOnlyModelViewSet):
     """
     queryset = LeaderboardEntry.objects.filter(user__visible=True)
     serializer_class = LeaderboardEntrySerializer
-    permission_classes = [permissions.IsAuthenticated]
+    permission_classes = [permissions.AllowAny]
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)